basic/src/test/java/org/jboss/hal/testsuite/test/homepage/NeedHelpTestCase.java

```diff
@@ -85,6 +91,21 @@ public void getHelpTest() throws Exception {
         assertLinksArePresent(section, links);
     }
 
+    @Test
+    public void newToEAP7Test() {
+        WebElement root = homePage.getContentRoot();
+
+        assertElementIsPresent("'New to EAP?' text should be present.", root,
+                new ByJQuery(":contains('" + PropUtils.get("homepage.newtoeap.text") + "')"));
```

Comment:
Please add related rows `homepage.newtoeap.text` and `homepage.newtoeap.link` to `common/src/main/resources/label.properties` as well ;-)